style.css

```diff
@@ -0,0 +1,45 @@
+body {
+    font-family: "Arial", "Helvetica Neue", "Helvetica, sans-serif";
+    color: #777;
+    background-color: #777;
+    width: 960px;
+    font-size: 18px;
+}
+#top {
+    background-color: #ebebeb;
+    padding: 20px;
+    margin-bottom: 20px;
+}
+#nav {
+    background-color: #ebebeb;
+    margin-top: 10px;
+    padding: 10px;
+    margin-bottom: 20px;
+}
+#section{
+    background: #ebebeb;
+    float: left;
+    width: 480px;
+    height: 200px;
+    padding: 40px;
+    clear: both;
+}
+#aside{
```

Comment:
I tried to add margins around your aside div but it didn't work for some reason.